packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/duration-fields.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import classNames from 'classnames';
+import { InputField, SelectField } from '.';
+import { TDropdownItems, TSelectsFieldNames } from '../quick-strategy.types';
+import { TDurationFields } from './components.types';
+
+const DurationFields = ({
+    id,
+    is_mobile,
+    field_name,
+    idx,
+    dropdown_list,
+    selected_value,
+    label,
+    select_value,
+    setFieldValue,
+    className,
+    is_able_disabled,
+    values,
+    onChangeDropdownItem,
+    onHideDropdownList,
+    onScrollStopDropdownList,
+    selected_trade_type,
+    selected_symbol,
+    handleChange,
+    onChangeInputValue,
+    setCurrentFocus,
+    errors,
```

Comment:
can we sort it, variable values on top and functions at the bottom?

Reply:
done ✅